src/site/content/en/blog/layoutNG/index.md

```diff
@@ -0,0 +1,234 @@
+
+#### Joining across Element Boundaries
+In some scripts graphemes join with adjacent ones and changes presentation. In LayoutNG this works even if the graphemes are in different elements, allowing the joins to be preserved even if different styling is applied ([#6122](https://crbug.com/6122)).
+
+{% Aside 'key-term' %}
+  a grapheme is the smallest unit of a writing system of any given language
+{% endAside %}
+
+The example below shows the rendering of the following HTML in legacy layout and LayoutNG respectively:
+
+```html
+<div>&#1606;&#1587;&#1602;</div>
+<div>&#1606;&#1587;<span>&#1602;</span></div>
+```
+
+<div class="w-figures-2-up">
+  <figure class="w-figure w-figure--center">
+    <img loading="lazy" src="legacy_shape.png" alt="proper grapheme on left and separated improper rendering on right">
+    <figcaption class="w-figcaption w-figcaption--center">
+      <small>Fig 3a, Legacy Layout</small><br>
+      Note how the form of the second letter changes
+    </figcaption>
+  </figure>
+
+  <figure class="w-figure w-figure--center">
+    <img loading="lazy" src="ng_shape.png" alt="proper combined graphemes shown">
+    <figcaption class="w-figcaption w-figcaption--center">
+      <small>Fig 3b, LayoutNG</small><br>
+      Note they're now identical
+    </figcaption>
+  </figure>
+</div>
+
+#### CJK Ligatures
+Although Chromium already supports ligatures and enable them by default, there are some limitations. Ligatures involving multiple CJK codepoints are not supported in the legacy engine due to a rendering optimization. LayoutNG removes these restrictions and supports ligatures regardless of script.
+
+The example below shows the rendering of three discretionary ligatures using the Adobe SourceHanSansJP font:
+
+<div class="w-figures-2-up">
+  <figure class="w-figure w-figure--center">
+    <img loading="lazy" src="legacy_dlig_jp.png" alt="middle character combination not forming ligature">
+    <figcaption class="w-figcaption w-figcaption--center">
+      <small>Fig 4a, Legacy layout</small><br>
+      MHz correctly forms a ligature<br>
+      but マンション and 10点 does not
+    </figcaption>
+  </figure>
+
+  <figure class="w-figure w-figure--center">
+    <img loading="lazy" src="ng_dlig_jp.png" alt="proper ligatures shown">
+    <figcaption class="w-figcaption w-figcaption--center">
+      <small>Fig 4b, LayoutNG</small><br>
+      All three groups form ligatures as expected
+    </figcaption>
+  </figure>
+</div>
+
+#### Size to Content
+For elements that size-to-content (such as inline blocks) the current layout engine computes the size of the block first and then performs layout on the content. In some cases, such as when a font kerns aggressively, this may result in a mismatch between the size of the content and the block. In LayoutNG this failure mode has been eliminated as the block is sized based on the actual content.
```

Comment:
```suggestion
For elements that size to content (such as inline blocks), the legacy layout engine computes the size of the block first and then lays out the content. In some cases, such as when a font has tight kerning, this may result in a mismatch between the size of the content and the block. In LayoutNG this failure has been eliminated by basing the block size on the actual content.
```
I thought "tight kerning" might be more recognizable than "aggressively kerned," assuming that's what was meant. Also, do we need to define "kerning"?

Reply:
hm, lemme think about this one and review again tomorrow
i dont think kerning needs defined (but i could be corrected)
i assume aggressive kerning was a purposeful choice, while i agree tight kerning is the general nomenclature

Reply:
I don't have strong opinions either. I'll defer to you about our audience's background knowledge.